tests/engine/ui/snapshot/testStory.js

```diff
@@ -0,0 +1,58 @@
+if (typeof PATH_TO_THE_REPO_PATH_UTILS_FILE === 'undefined') PATH_TO_THE_REPO_PATH_UTILS_FILE = "https://raw.githubusercontent.com/highfidelity/hifi_tests/master/tests/utils/branchUtils.js";
+Script.include(PATH_TO_THE_REPO_PATH_UTILS_FILE);
+var nitpick = createNitpick(Script.resolvePath("."));
+Script.include(nitpick.getUtilsRootPath() + "test_stage.js");
+
+nitpick.perform("Test snapshot with no snap directory set", Script.resolvePath("."), "secondary", function(testType) {
+    
+    var oldSnapshotsLocation;
+    
+    function cleanup() {
+        if (oldSnapshotsLocation != undefined) {
+            Snapshot.setSnapshotsLocation(oldSnapshotsLocation);
+            oldSnapshotsLocation = undefined;
+        }
+    }
+    
+    Script.scriptEnding.connect(cleanup);
+    
+    function clearSnapshotsLocation() {
+        if (oldSnapshotsLocation == undefined) {
+            oldSnapshotsLocation = Snapshot.getSnapshotsLocation();
+        }
+        Snapshot.setSnapshotsLocation("");
+    }
+    
+    nitpick.addStep("Primary Camera Snapshot", function () {
+        clearSnapshotsLocation();
+        Window.takeSnapshot();
+    });
+    
+    nitpick.addStep("Primary Camera Snapshot and GIF", function () {
+        clearSnapshotsLocation();
+        Window.takeSnapshot(false, true);
```

Comment:
All of the other calls use notify = true.  I imagine it doesn't matter much, but maybe we should be consistent?

Reply:
Yea, might as well.